weave

```diff
@@ -1489,7 +1507,7 @@ launch_router() {
         --dns-effective-listen-address $DOCKER_BRIDGE_IP \
         ${NETHOST_OPT:+$DNS_ROUTER_OPTS} $NO_DNS_OPT \
         --http-addr $HTTP_IP:$HTTP_PORT \
-        --docker-api "unix:///var/run/docker.sock" "$@")
+        "$@")
```

Comment:
This change, in conjunction with the new default value for `dockerAPI` in `prog/weaver/main.go`, means that if the user does _not_ have `DOCKER_HOST` set in their environment when they launch the router then it will silently fail to subscribe to container events and hence all the IPAM & DNS tombstoning magics will not be invoked...